packages/@aws-cdk/aws-appmesh/test/virtual-node.test.ts

```diff
@@ -478,8 +470,51 @@ describe('virtual node', () => {
             ],
           },
         });
+      });
+
+      test('should add a backend virtual service to the resource without using a CDK Ref', () => {
```

Comment:
This name could use some work I think:

```suggestion
      test('you can add a Virtual Service as a backend to a Virtual Node which is the provider for that Service', () => {
```